jetstream/tests/test_bigquery_client.py

```diff
@@ -7,18 +7,18 @@
 
 def test_delete_experiment_tables():
```

Comment:
Not worth worrying about in this PR, and I might be missing an important reasoning for the current structure, but it feels like this test should be structured as a class with a common setup function and test cases for each of the sections in this function instead of resetting the mocks between tests.